package/yast2-crowbar.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-crowbar
-Version:        4.0.0
+Version:        4.0.1
```

Comment:
Just a note about the version: Will be the SOC9 based on SLE15-GA or SLE15-SP1?

The YaST versioning scheme for SP1 is 4.1.x, it would be nice use the same versioning scheme here as well. (See the detail in the [documentation](https://yastgithubio.readthedocs.io/en/latest/versioning/#new-schema-version-numbers-are-related-to-suse-versions).)

Reply:
SLE12SP4 actually ... 
Our previous SOC has `yast2-crowbar-3.2.0` , so should I go back to 3.2.X versioning instead?
(Seems that unfortunatelly we do not have our latest change tracked in git...)

Reply:
No, would not go back, this could cause some troubles when upgrading the package. Just keep it as it is now, the number does matter much...

Reply:
I'm actually fine with doing `3.4.0` version, and do not target the change to `master`, but to some older branch.